src/fsharp/FSStrings.resx

```diff
@@ -970,7 +970,7 @@
     <value>This rule will never be matched</value>
   </data>
   <data name="ValNotMutable" xml:space="preserve">
-    <value>This value is not mutable</value>
+    <value>Values are immutable by default. If you wish to mutate this value, you must explicitly declare it as a mutable variable using the mutable keyword, e.g. 'let mutable {0} = expression'.</value>
```

Comment:
One small modification please:

    Values are immutable by default. If you intend to to mutate this value, declare it using the mutable keyword, e.g. 'let mutable {0} = expression'.

thanks